pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -200,6 +200,9 @@ type JaegerQuerySpec struct {
 
 	// +optional
 	JaegerCommonSpec `json:",inline,omitempty"`
+
+	// +optional
```

Comment:
Could you please leave a simple comment with the possible values? If you don't want to list them all, you can point the user to the right direction (like the Kubernetes' Service documentation page).